templates/solidity-frameworks/hardhat/packages/hardhat/hardhat.config.ts.template.mjs

```diff
@@ -41,6 +41,7 @@ const config: HardhatUserConfig = {
     },
   },
   networks: {
+    ${networks[0] && `${networks},`}
```

Comment:
nit: for me && \`${networks[0]}\` is better. `${networks}` works, but it converts ["something"] to "something", not so obvious  (same for `extendTheme`)

Reply:
<img width="251" alt="image" src="https://github.com/user-attachments/assets/7b5b5026-913f-48d7-9b0c-5a78d9fb4d34">

and in case of multiple elements in the array they will by stringified, not sure it's desired behavior, I think one of them should be chosen

Reply:
Yes, I wasn't sure because of the multiple extensions thing. In some cases override is the only choice, like solidityVersion....but in "networks" for example it would make sense to have multiple, right?

Anyway, I think this is a bigger conversation to have cc @technophile-04 :
- We should remove arrays if only 1 extension is allowed
- If we allow multiple, see how we handle multiple extensions adding the same variable 

For now, assuming it's only 1 extension. https://github.com/scaffold-eth/create-eth/pull/109/commits/2bcc90201f39aec97eb1348c6e29d5971bc6e15a

Thanks for all the feedback Rinat!

Reply:
> In some cases override is the only choice, like solidityVersion....but in "networks" for example it would make sense to have multiple, right?
> 
> Anyway, I think this is a bigger conversation to have cc @technophile-04 :


Yes need to think about it nicely. Will create an issue for this later. 

Btw we are already kind of leveraging multiple extensions for example when you choose ([hardhat](https://github.com/scaffold-eth/create-eth/blob/main/templates/solidity-frameworks/hardhat/packages/nextjs/scaffold.config.ts.args.mjs) + [onchainkit extension](https://github.com/scaffold-eth/create-eth-extensions/blob/onchainkit/extension/packages/nextjs/scaffold.config.ts.args.mjs)) on this line: 


https://github.com/scaffold-eth/create-eth/blob/faa6dabd44d876406eaab01c208808ccbaf68607/templates/base/packages/nextjs/scaffold.config.ts.template.mjs#L16

Also another nitpicky thing which I wanted to mention is the variable names, like there are two sides of it to look at

on create-eth side in `template.mjs` it makes sense to have them pural for eg: `chainName` => `chainNames`

but if you look from extension developers side it should be singular `chainName` without s. 

I think maybe we should give priority to extension side? and always use singular so that for extensions developers it makes sense for them? 

^ we have some inconsistency on this naming in codebase lol just wanted to mention since it was in back of mind.